### ISSUE TITLE
Why the last conv block dose not have batch normalization?

### ISSUE THREAD

**Issue by kevinbro96** [kuc2477/pytorch-vae#1]
Hi, there ! I found that the last conv block of both encoder and decoder do not have batch normalization. What is the reason for this setting?
